.travis.yml

```diff
@@ -128,9 +124,11 @@ install:
     - if [[ $GINGA_VERSION == stable ]]; then pip -q install ginga; fi
 
 script:
+   - pip install tox tox-pypi-filter --upgrade
+   - pip install -e .
    - $MAIN_CMD $SETUP_CMD
 
 after_success:
-    # If coveralls.io is set up for this package, uncomment the line below.
-    # The coveragerc file may be customized as needed for your package.
-    # - if [[ $SETUP_CMD == *coverage* ]]; then coveralls --rcfile='astrowidgets/tests/coveragerc'; fi
+    # Upload coverage to codecov
+    - pip install codecov
+    - codecov
```

Comment:
Note: Need to double check that it uploads properly for the run after the merge of this PR.